src/job_cache/daemon_cache.cpp

```diff
@@ -945,8 +945,11 @@ FindJobResponse DaemonCache::read(const FindJobRequest &find_request) {
   std::string msg = cmd.serialize();
   msg += '\0';
 
+  wcl::log::info("Sending Read command to eviction loop")();
```

Comment:
Super long term idea not for this PR, but it would be a cool optimization to hash static messages then emit the hash everywhere we see the message + a message->hash header. Then the log viewer could reconstruct the actual log